taipy/core/data/_data_manager.py

```diff
@@ -38,11 +39,33 @@ def _get_or_create(
         scope = data_node_config.scope
         parent_id = pipeline_id if scope == Scope.PIPELINE else scenario_id if scope == Scope.SCENARIO else None
 
-        if dn_from_parent := cls._repository._get_by_config_and_parent_ids(data_node_config.id, parent_id):
+        if dn_from_parent := cls._repository._get_by_config_and_parent_id(data_node_config.id, parent_id):
             return dn_from_parent
 
         return cls._create_and_set(data_node_config, parent_id)
 
+    @classmethod
+    def _get_or_creates(
```

Comment:
get_or_creates is not good to me. 
Rename to _get_or_create_multiple or something like that.

Reply:
bulk_get_or_create ? batch_get_or_create ?

Reply:
yes why not.

Reply:
This created some inconsistency between datamanager and the rest of the managers (datamanager doesn't have get_or_create). I dont think it's a problem, but think it's something to be mindful about to make sure the code is clean and consistent :D